lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -1227,24 +704,6 @@ module Resources
                   }
                 ],
                 'text': 'Default Guarantor'
-              },
-              {
-                'id': 'CI-97982588-1-0',
-                'coding': [
-                  {
-                    'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/40',
-                    'code': '670847',
-                    'display': 'Child',
-                    'userSelected': true
-                  },
-                  {
-                    'system': 'http://terminology.hl7.org/CodeSystem/v3-RoleCode',
-                    'code': 'CHILD',
-                    'display': 'child',
-                    'userSelected': false
-                  }
-                ],
-                'text': 'Child'
```

Comment:
This needs to be added back as well.

Reply:
added back https://github.com/cerner/fhir.cerner.com/pull/624/commits/0105b4346c586ef01f5a2008435fd01408993d76#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR709-R726